### PR TITLE
Completed html-form Jimmy Tran

### PR DESCRIPTION
**Pull request by JimmyTran1206**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          <option value="">Because it's powerful</option>
+        </select>
+      </div>
+      &nbsp;
+
+      <div>
+        <label for="radio">Favorite Language (choose one)</label><br />
+        <input type="radio" name="radButton" id="rad1" /> HTML
+        <input type="radio" name="radButton" id="rad2" /> CSS
+        <input type="radio" name="radButton" id="rad3" /> JavaScript
```
Comment (shawnkost) [inline html-forms/index.html:33]: Each one of these `input`s should have their own `label`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          <option value="">Select an option</option>
+          <option value="">Because it's interesting</option>
+          <option value="">Because it's fun</option>
+          <option value="">Because it's powerful</option>
+        </select>
+      </div>
+      &nbsp;
+
+      <div>
+        <label for="radio">Favorite Language (choose one)</label><br />
```
Comment (shawnkost) [inline html-forms/index.html:30]: This should be a `p` element rather than a `label`. This `label` isn't connected to any form control elements

**Review by shawnkost — CHANGES_REQUESTED** [review]
A couple changes here Jimmy!

File: html-forms/index.html (hunk 1 above)
Comment (JimmyTran1206) [reply html-forms/index.html]: thank you Shawn. I have put the controls in labels according to your instruction

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**